docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Trigger Cloud Sync conditionally"
+linkTitle: "Trigger Sync"
+description: "Configure cloud sync to automatically capture data in the Viam app."
+weight: 36
+type: "docs"
+tags: ["data management", "cloud", "sync"]
+# SME: Alexa Greenberg
+---
+
+If you rely on mobile data but have intermittent WiFi connection in certain locations or at certain times of the day, you may want to trigger sync to only occur when these conditions are met.
+
+This module allows you to configure Cloud Sync to occur only at a specific time frame by implementing a sensor, `naomi:sync-at-time:timesyncsensor`, that enables sync when within a specified timeframe and disables sync when outside that timeframe.
```

Comment:
```suggestion
This module allows you to configure Cloud Sync to occur only at a specific time frame by implementing a sensor, `naomi:sync-at-time:timesyncsensor`, that enables sync within a specified time frame and disables sync outside that time frame.
```
[opt/nits] I think the "when"s are awk and not necessary.
Consider either caps or not for cloud sync, and either time frame or timeframe throughout.